File: pocketr-java.api/src/test/java/com/decrux/pocketr/api/ApiApplicationTests.java
package com.decrux.pocketr.api;

import org.junit.jupiter.api.Disabled;
import org.junit.jupiter.api.Test;
import org.springframework.boot.test.context.SpringBootTest;
import org.springframework.test.context.ActiveProfiles;

@SpringBootTest
@ActiveProfiles("test")
@Disabled("Temporarily disabled until a PostgreSQL test database is provisioned in CI/local test environment")
class ApiApplicationTests {

    @Test
    void contextLoads() {
    }
}

File: pocketr-api/src/test/kotlin/com/decrux/pocketr/api/ApiApplicationTests.kt
package com.decrux.pocketr.api

import com.decrux.pocketr.api.testsupport.UsePostgresDb
import org.junit.jupiter.api.Disabled
import org.junit.jupiter.api.Test
import org.springframework.boot.test.context.SpringBootTest

@SpringBootTest
@UsePostgresDb
@Disabled("Temporarily disabled until a PostgreSQL test database is provisioned in CI/local test environment")
class ApiApplicationTests {
    @Test
    fun contextLoads() {
    }
}
